oap-server/server-storage-plugin/storage-influxdb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/influxdb/InfluxStorageConfig.java

```diff
@@ -32,6 +32,7 @@
 
     private int actions;
     private int duration;
+    private boolean batchEnabled = true;
```

Comment:
I believe you need to update the `application.yml` otherwise how could this take effect?

Reply:
https://github.com/apache/skywalking/blob/127782f135223c2741c7c65a9d9386c49a8aa1be/oap-server/server-bootstrap/src/main/resources/application.yml#L187-L196

Reply:
ok,very thanks